src/style.css

```diff
@@ -1,3 +1,64 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
It's good practice to include a fallback font in case the specified font doesn't load. You can add a generic font family such as 'sans-serif' as a fallback. [Explanation](https://www.w3schools.com/cssref/pr_font_font-family.asp)